README.md

```diff
@@ -86,3 +87,49 @@ Refer to the [Bounty Board Commands and Workflow](https://bankless.notion.site/T
 6. **Frontend Status: In Review**: Card status on the frontend is "In Review".
 7. **Discord/Bot: Complete**: Bounty claimer can signal completion ✅ on the post in the #🧀-bounty-board channel. Bounty creator receives message to tip bounty completer.
 8. **Frontend Status: Completed**: Work is done.
+
+
+# Getting Started
+In order to run a local instance of the application you will need to configure a few things.
+### Install Packages
+You can either run the application from this top-level monorepo, or change into the `packages/react-app` directory. You will need the yarn package manager installed then run
+
+```bash
+$ yarn
+$ yarn dev 
+```
+Which will install packages and run the application on port 3000. The `package.json` file in the respective repo gives a full list of commands.
+### Configure local `.env` file
+
+The react application looks for an environment variables file named `.env.local` on starting. You can copy the `.env.qa` file for most of the variables. You will need to add the `DEV_MONGODB_URI` and `DEV_DEV_DISCORD_BOUNTY_BOARD_WEBHOOK` as covered below.
+### Setup MongoDB
+
+Connection to MongoDB is handled through the Mongoose DRM. You can either connect to a hosted instance of MongoDB, or run a local development copy.
+
+If running locally, your `.env.local` will contain something like:
+```
+DEV_MONGODB_URI=mongodb://localhost:27017/bountyboard
```

Comment:
We've made a migration from netlify to vercel which allows us to store environments grouped by code level (production, deployment, preview). As such we've renamed the environment variable DEV_MONGODB_URI ->

MONGODB_URI=mongodb://localhost:27017/bountyboard

Reply:
Cool, have made changes as requested into the next commit in the same PR